quinn-proto/src/connection/mod.rs

```diff
@@ -796,6 +796,37 @@ impl Connection {
                 break;
             }
 
+            // Send an off-path PATH_RESPONSE
+            if space_id == SpaceId::Data && num_datagrams == 1 {
```

Comment:
So AIUI this means we're always yielding off-path `PATH_RESPONSE` frames before yielding any on-path packets. Is that optimal? I would (maybe naively) expect that we might want to prioritize on-path packets and only then yield off-path challenge responses in order to minimize latency for the former?

Reply:
This prioritization is intended. Added an explanatory comment.